prow/getting_started.md

```diff
@@ -32,8 +32,8 @@ that has read and write access to the bot account. Generate it from the
 [account's settings -> Personal access tokens -> Generate new token][1].
 
 ```sh
-kubectl create secret generic hmac-token --from-file=hmac=/path/to/hook/secret
-kubectl create secret generic oauth-token --from-file=oauth=/path/to/oauth/secret
+kubectl create secret generic hmac-token --from-file=hmac=/path/to/hook/hmac
```

Comment:
Eh, the previous "path" was a sentence -- `"path to hook secret"` -- so maybe let's make the change to a different but valid sentence like `path/to/hmac/token`. `"path to hook hmac"` isn't really valid.

Reply:
I think the idea was to make the file names unique to help prevent someone from accidentally creating a secret incorrectly. If we did `/path/to/hmac/token` and `/path/to/oauth/token` the files names would both be `token`.
Maybe do `/path/to/hmac-secret` and `/path/to/oauth-secret`?
/shrug

Reply:
The idea was to make sure the file name matches the referenced file.
--from-file=hmac=/path/to/hook/hmac
as it can be easily overlooked and in the current format the following would happen:
--from-file=/path/to/oauth/secret (name is secret instead of hmac)

> Maybe do /path/to/hmac-secret and /path/to/oauth-secret?
This is possible as well, but we'd have to change the deployments in that case.

Reply:
SGTM.
That being said, I think many users will create their own temp files with different names to store the secrets in which case the same problem would occur if the data key is omitted. So it really is best if they just copy and paste the commands and modify the file path as needed. We shouldn't spend too much time optimizing for this type of misuse.